src/core/transactionmanager.js

```diff
@@ -520,6 +521,17 @@ TransactionManager.prototype._resubmit = function(ledgers_, pending_) {
 TransactionManager.prototype._prepareRequest = function(tx) {
   const submitRequest = this._remote.requestSubmit();
 
+  if (tx.hasMultiSigners()) {
+    tx.setSigningPubKey('');
+
+    if (this._remote.local_signing) {
+      tx.tx_json.Signers = tx.getMultiSigners();
+    } else {
+      submitRequest.message.command = 'submit_multisigned';
+      submitRequest.message.Signers = tx.getMultiSigners();
+    }
+  }
+
   if (this._remote.local_signing) {
     tx.sign();
```

Comment:
I think I may understand what's going on.  The decision between regular- and multi-signing is based on the state of the `SigningPubKey` field.  If the field has a non-empty value then we're regular signing.  If the field is empty then we're multi-signing.  I have a unit test that looks for a `temINVALID` if the `SigningPubKey` is non-empty but `Signers` is present.

I don't (yet) have a test with an empty `SigningPubKey` and both `Signers` and `TxnSignature` filled in.  I bet that is the case you're seeing.

The good news is that the `SigningPubKey` value determines which kind of signing we do.  If it's non-empty it's regular signing.  If it's empty we're multisigning.  So the bug could have been worse.  Still, I'll try to figure out a way to wrangle the unit test framework into performing this test.  Assuming the test fails then I'll fix the bug.  If the test doesn't fail then I'll still be puzzled.